test/schema.json

```diff
@@ -397,6 +397,11 @@
                 }
               }
             }
+          },
+          "groupHotkeys": {
```

Comment:
I am aware that the schema currently isnt very strict with properties in most of its parts, but I am planning on updating it in the future. The PR looks good so far, so the only change needed is to add explicit property keys for the groupHotkeys object with their types and to disallow additional keys to be present.

Reply:
Going to merge it anyway today and update the schema myself afterwards to get the release going with all the pending changes. Thank you for adding the feature!